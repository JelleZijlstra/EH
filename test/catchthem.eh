#!/usr/bin/ehi
try
	$print 3
catch
	printvar exception
finally
end
