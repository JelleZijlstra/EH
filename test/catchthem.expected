%%stdout%%
@object <UnknownCommandError> [
command <public,non-static,non-constant>: @string 'print'
]
%%stderr%%
